File: ts-sdk/src/sdk/models/operations/completeanerroreddeploy.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class CompleteAnErroredDeployPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class CompleteAnErroredDeployRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CompleteAnErroredDeployPathParams;
}

export class CompleteAnErroredDeployResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  completeAnErroredDeploy200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/completeanerroreddeploy.go
package operations

type CompleteAnErroredDeployPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type CompleteAnErroredDeployRequest struct {
	PathParams CompleteAnErroredDeployPathParams
}

type CompleteAnErroredDeployResponse struct {
	ContentType                                     string
	StatusCode                                      int
	CompleteAnErroredDeploy200ApplicationJSONObject map[string]map[string]interface{}
}
